scheduler/crates/api/src/shared/auth/policy.rs

```diff
@@ -76,77 +76,68 @@ mod test {
     fn permissions() {
         let policy = Policy::default();
         assert!(policy.authorize(&Vec::new()));
-        assert!(!policy.authorize(&vec![Permission::CreateCalendar]));
+        assert!(!policy.authorize(&[Permission::CreateCalendar]));
```

Comment:
No need to use a Vector
https://rust-lang.github.io/rust-clippy/master/index.html#/useless_vec